lib/wovnrb/store.rb

```diff
@@ -185,14 +178,47 @@ def wovn_host
     end
 
     private
-    def cleanSettings
-      @settings['ignore_globs'] = []
-    end
 
     def stringify_keys!(h)
       h.keys.each do |k|
         h[k.to_s] = h.delete(k)
       end
     end
   end
+
+  class Settings < Hash
+    def initialize(*args, **kwargs)
+      super(*args, **kwargs)
+      @dynamic = {}
+    end
+
+    def [](key)
+      return @dynamic[key] if @dynamic.key?(key)
+      return ignore_globs if key == 'ignore_globs'
+      super(key)
+    end
+
+    def ignore_globs
+      ignore_paths = self['ignore_paths']
+      return [] unless ignore_paths.kind_of?(Array)
+      ignore_paths.map { |pattern| Glob.new(pattern) }
+    end
+
+    def clear_dynamic!
+      @dynamic.clear
+    end
+
+    def update_dynamic!(params)
+      # If the user defines dynamic settings for this request, use it instead of the config
+      DYNAMIC.each do |params_key, setting_key|
+        value = params[params_key]
+        @dynamic[setting_key] = value if value
+      end
+    end
+
+    DYNAMIC = {
```

Comment:
please not use the same name in same scope (`@dynamic` and `DYNAMIC`)

Reply:
It's renamed `DYNAMIC_KEYS`.
Thanks.